packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -208,13 +208,31 @@ regen=--regen
 # shellcheck disable=SC2034
 regen=
 
+#################################
+# Tests to symbolically execute #
+#################################
+# Missing: OptimismPortalKontrol.prove_proveWithdrawalTransaction_paused
+test_list=( "OptimismPortalKontrol.prove_finalizeWithdrawalTransaction_paused" \
+             "L1StandardBridgeKontrol.prove_finalizeBridgeERC20_paused" \
+             "L1StandardBridgeKontrol.prove_finalizeBridgeETH_paused" \
+             "L1ERC721BridgeKontrol.prove_finalizeBridgeERC21_paused" \
+             "L1CrossDomainMessengerKontrol.prove_relayMessage_paused"
```

Comment:
nit
```suggestion
test_list=( "OptimismPortalKontrol.prove_finalizeWithdrawalTransaction_paused" \
            "L1StandardBridgeKontrol.prove_finalizeBridgeERC20_paused" \
            "L1StandardBridgeKontrol.prove_finalizeBridgeETH_paused" \
            "L1ERC721BridgeKontrol.prove_finalizeBridgeERC21_paused" \
            "L1CrossDomainMessengerKontrol.prove_relayMessage_paused"
```

Reply:
Strangely, this is the formatting my editor follows

Reply:
I don't have `shfmt` or similar installed so just assumed you were trying to line up the spaces. If this is how the formatter leaves it that seems ok to me

Reply:
This is emacs' `Shell-script` mode indenting like that, but I'm fine with having them aligned. I actually raised an eyebrow when I saw this indentation